packages/ui-extensions/src/surfaces/admin/components/Link/examples/AbsoluteLink.example.ts

```diff
@@ -1,13 +0,0 @@
-import {extension, Link} from '@shopify/ui-extensions/admin';
-
-export default extension('Playground', (root) => {
-  const link = root.createComponent(Link, {
```

Comment:
I think we could revise this example and call external-link.example.ts. It's still useful to signal that external links must be absolute links and will be opened in a new window/tab